apps/leo_manager/src/leo_manager_formatter_text.erl

```diff
@@ -1099,14 +1099,13 @@ bucket_by_access_key(Buckets) ->
                   BucketStr = binary_to_list(Bucket1),
                   PermissionsStr = leo_s3_bucket:aclinfo_to_str(Permissions1),
 
-                  RedOptions = leo_redundant_manager_api:get_options(),
+                  {ok, RedOptions} = leo_redundant_manager_api:get_options(),
```

Comment:
As https://github.com/leo-project/leofs/pull/696 already merged,
Please remove this line.